index.bs

```diff
@@ -1626,8 +1626,7 @@ Requirements for All Working and Interest Groups</h3>
 	The Director appoints (and re-appoints) Chairs for all groups.
 	The Chair is a <a href="#member-rep">Member representative</a>,
 	a <a href="#Team">Team representative</a>,
-	or an <a href="#invited-expert-wg">Invited Expert</a>
-	(invited by the Director).
```

Comment:
I don't think this one is necessarily gratuitous.  Invited Experts generally need to be invited by the Chair, do they not?  But this is to INVITE the chair, so it needs to be clear who has that authority.

Reply:
You are correct. I got confused because the Director can rescind the invitation of an IE, and that somehow made me think that IE's invitation always came from the Director. But they do not, so this change is incorrect. Will revert.